packages/node-app/src/utils/auth.utils.ts

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+
+import { Unit } from '../models/user.model';
+import TAClient from './ta.utils';
+
+const googleAuthInstance = axios.create({
+  baseURL: process.env.GOOGLE_OAUTH_URI,
+});
+
+const getUserInfo = async (tokenType, accessToken) => {
+  try {
+    const res = await googleAuthInstance.get('/userinfo', {
+      headers: {
+        Authorization: `${tokenType} ${accessToken}`,
+        Accept: 'application/json',
+      },
+    });
+
+    if (res.status === 200) {
+      const userInfo = res.data;
+      const taSession = await TAClient.signIn();
+      const runtimePeople = await TAClient.getPeople(taSession, false);
```

Comment:
Isn't there an endpoint to get only the information of the current user?

Reply:
There is one, but does not provide the information that we need, such as skills and archived